app/views/collections/index.rss.builder

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+# Helper method for the collections description
+# derived from the associated landing page strapline.
+def strapline(collection)
```

Comment:
Let's move this into `CollectionsHelper` as `collection_strapline`, then make both this RSS view and the `Collections::Show` Mustache view class use it, to keep it DRY, removing the assignment of `@total_item_count` from `CollectionsController#show`.